### ISSUE TITLE
Multiple warnings when capturing an image

### ISSUE THREAD

**Issue by tilowestermann** [jim-easterbrook/python-gphoto2#12]
I'm using the latest version go python-gphoto2 and experiencing the following warnings when capturing a picture (Canon 600D) on a Raspberry Pi 2:
```
WARNING: gphoto2: (foreach_func [gphoto2-port-info-list.c:237]) Error during assembling of port list: 'Unspecified error' (-1).
WARNING: gphoto2: (ptp_list_folder_eos [library.c:7017]) storage 0xffffffff, but handle 0x00000000?
Capturing image
WARNING: gphoto2: (camera_canon_eos_capture [library.c:3088]) no focus info?
Camera file path: /store_00020001/DCIM/100CANON/IMG_3223.JPG
Copying image to /tmp/IMG_3223.JPG
WARNING: gphoto2: (ptp_usb_getresp [usb.c:428]) PTP_OC 0x911c receiving resp failed: PTP Device Busy (0x2019)
WARNING: gphoto2: (camera_unprepare_canon_eos_capture [config.c:450]) 'ptp_canon_eos_resetuilock (params)' failed: PTP Device Busy (0x2019)
```

Using the command line, there is no warning. 

Installed gphoto2 info:
```
gphoto2         2.5.9          gcc, popt(m), exif, no cdk, no aa, jpeg, no readline
libgphoto2      2.5.9          all camlibs, gcc, ltdl, EXIF
libgphoto2_port 0.12.0         gcc, ltdl, USB, serial without locking
```

**Comment by jim-easterbrook** [comment]
Do you get the same messages if you run the gphoto2 command with `--debug-loglevel error --debug` options?
Libgphoto2 doesn't discriminate well enough between errors, warnings and debug info in my opinion. You either get nothing or a lot of stuff that isn't really a problem. You could try fine-tuning the logging as described in the python-gphoto2 readme.

**Comment by WillNye** [comment]
I'm seeing the same issue it also seems to have an impact on interval photos, it seems to be overwriting images on the camera as opposed to appending them. As a result this will take 15 photos but only 1 will be on the camera (the last photo taken)

```
    for x in range(30):
        file_path = gp.check_result(gp.gp_camera_capture(
            camera, gp.GP_CAPTURE_IMAGE, context))
    gp.check_result(gp.gp_camera_exit(camera, context))
```

**Comment by jim-easterbrook** [comment]
Have you set the capture target to "memory card" or "internal ram"? (Those are the options on my camera, yours may be different.)
Do you get a different `file_path` value on each capture?

**Comment by WillNye** [comment]
I finally figured it out after banging my head against a wall. I had set it manually to the memory card, HOWEVER, it was getting reset to write to RAM after the first photo was taken. The reason turns out to be my camera was plugged into a USB port which was not integrated. After plugging into a USB port that was directly on the MOBO it works fine now. A very odd bug and no clue what causes this. I tried it on 5 different external ports and got the same issue. Plugged it into 3 MOBO USB ports works like a charm.
